src/guides/v2.3/coding-standards/docblock-standard-general.md

```diff
@@ -555,13 +555,16 @@ interface MutableInterface
 {
     /**
      * Returns 0, if no value is available
+     *
+     * @return int
      */
     public function getVal(): int;
 
     /**
      * Sets 0 in case a non-integer value is passed
      *
      * @param int|string|bool|float|null $value
+     * @return void
```

Comment:
It contradicts to the [Functions and methods section](https://devdocs.magento.com/guides/v2.3/coding-standards/docblock-standard-general.html#functions-methods) 
> The declaration of the return type using the @return tag must only be added if the method return type signature does not supply all necessary information

Reply:
@lenaorobei Thanks for the feedback. I have removed `@return` tag in PHPDoc. But I have checked few files in Magento Modules, I can able to see `@return` tag & method return type signature for same method. ex: [link](https://github.com/magento/magento2/blob/2.4-develop/app/code/Magento/Cms/Model/Page/CustomLayout/Data/CustomLayoutSelectedInterface.php). is this valid?